src/main/java/codeu/controller/ProfilePagesServlet.java

```diff
@@ -1,19 +1,33 @@
 package codeu.controller;
 
+import codeu.model.data.Conversation;
+import codeu.model.data.Message;
 import codeu.model.data.User;
+import codeu.model.store.basic.ConversationStore;
+import codeu.model.store.basic.MessageStore;
 import codeu.model.store.basic.UserStore;
 import java.io.IOException;
+import java.time.Instant;
+import java.util.List;
+import java.util.UUID;
 import javax.servlet.ServletException;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import org.jsoup.Jsoup;
+import org.jsoup.safety.Whitelist;
 
 /** Servlet class responsible for the profile pages. */
 public class ProfilePagesServlet extends HttpServlet {
 
   /** Store class that gives access to Users. */
   private UserStore userStore;
 
+  /** Store class that gives access to Messages. */
+  private MessageStore messageStore;
+
+  private static final int USERNAME_INDEX = "/users/".length();
```

Comment:
I think you can just do 7 and then add a comment that says it evaluates to the length of "/users"